### PR TITLE
Show Store product page for Pro if company is has Pro subscription (not a Plan)

### PR DESCRIPTION
**Pull request by fjvallarino**
This is ready on: https://apps-beta.risevision.com/displays/details/YPD9TT7QWYHM?cid=ee5faa1a-3450-40d1-bb36-4d749cc8ef6f

@Rise-Vision/apps please review

### CODE REVIEW

File: src/templates/components/plans/plans-modal.html [hunk 1]
```diff
@@ -62,15 +62,15 @@ <h1>${{plan.priceMonth}}</h1>
         common-header.plans.learn-more-3
       </a>
     </div>
-    <div ng-show="showRPPLink && !playerProSubscriptionId" class="text-center u_margin-md-top">
+    <div ng-show="showRPPLink && !proSubscriptionLinkVisible()" class="text-center u_margin-md-top">
       <a class="btn btn-link btn-lg" target="_blank" href="https://store.risevision.com/product/2048" translate>
         common-header.plans.individual-licenses
       </a>
       <a class="cta_button btn btn-primary u_margin-lg" target="_blank" href="https://store.risevision.com/product/2048" translate>
```
Comment (alex-deaconu) [inline src/templates/components/plans/plans-modal.html]: @fjvallarino I know this is not a change from this PR, but I think both these links need the Company Id added. You may be able to use the 'link-cid' directive to add it in. Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino PR looks good, one suggestion for improvement inline. Thanks.

**Comment by fjvallarino** [comment]
@alex-deaconu updated!

**Comment by alex-deaconu** [comment]
@fjvallarino nice, thanks for fixing 👍

**Pull request merged**